ironic/resource_ironic_node_v1.go

```diff
@@ -289,6 +293,8 @@ func resourceNodeV1Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("network_interface", node.NetworkInterface)
 	d.Set("owner", node.Owner)
 	d.Set("power_interface", node.PowerInterface)
+	d.Set("properties", node.Properties)
+	d.Set("root_device", node.Properties["root_device"])
```

Comment:
An alternative to `DiffSuppressFunc` might be to just delete the key:

```golang
d.Set("root_device", node.Properties["root_device"])
delete(node.Properties, "root_device")
d.Set("properties", node.Properties)
```

Reply:
Hmm, I was thinking we'd want to reflect the value in the resource outputs for consistency, but I guess that's not really needed if we expose it in the properties

Reply:
The problem I want to avoid is terraform thinking the actual state of properties doesn't match the target state -- terraform re-reads the resource on several occasions and I suspect it will see that properties doesn't match what's in the HCL, if that makes sense.  Or worse, it'll explode on the nested data structure again